test/fixtures/test-helpers/input.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  app: {
+    'app.js': '// app',
+    'resolver.js': '// resolver'
+  },
+  config: {
+    'environment.js': '"ENV"'
+  },
+  tests: {
+    helpers: {
+      'resolver.js': 'import Resolver from "../../app/resolver";',
```

Comment:
So, @iezer that explains why you had to handle the double `app`. Can you update the PR?